ergo-wallet/src/main/scala/org/ergoplatform/wallet/secrets/EncryptedSecret.scala

```diff
@@ -24,4 +28,40 @@ object EncryptedSecret {
       Base16.encode(authTag), cipherParams)
   }
 
+  implicit object EncryptedSecretEncoder extends Encoder[EncryptedSecret] {
```

Comment:
Why adding this boilerplate -- encoder/decoder for this class is easily derived by circe?

Reply:
Unfortunately, that macros generates code not supported by native-image of GraalVM. 
This is the point of this PR, to make the code native-image compatible.